content-src/components/Spotlight/SpotlightFeed.scss

```diff
@@ -0,0 +1,129 @@
+$feed-item-border-color: #EEE;
+
+.grouped-highlight-feed {
+  ul {
+    line-height: 1.2;
+  }
+
+  .feed-item {
+    margin: 1rem 0;
+    border: 1px solid $feed-item-border-color;
+  }
+
+  .feed-item::before {
+    display: none;
+  }
+
+  .feed-item::after {
+    display: none;
+  }
+
+  .site-icon.feed-icon,
+  .feed-icon-image {
+    width: 96px;
+    height: 64px;
+  }
+
+  .feed-icon-image {
+    position: relative;
+    border: 1px solid $faintest-black;
+    border-radius: $border-radius;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    background-position: center;
+    background-size: cover;
+
+    flex-shrink: 0;
+
+    .site-icon.feed-icon {
+      position: absolute;
+      top: 50%;
+      left: 10px;
+      margin-top: -16px;
+    }
+  }
+
+  .feed-item > a {
+    padding: 0;
+  }
+
+  .feed-description {
+    margin-left: 24px;
```

Comment:
this is actually `$base-gutter`. Also, I would recommend using padding for spacing inside the element rather than margin